docs/fix/empty-wheelbarrows.rst

```diff
@@ -0,0 +1,33 @@
+fix/empty-wheelbarrows
+======================
+
+.. dfhack-tool::
+    :summary: Empties stuck items from wheelbarrows
+    :tags: fort bugfix items
+
+Empties all wheelbarrows which contain rocks that have become 'stuck' in them.
+
+This works around the issue encountered with :bug:`6074`, and should be run
+if you notice wheelbarrows lying around with rocks in them that aren't
+being used in a task.
+
+
+Usage
+-----
+
+``fix/empty-wheelbarrows [options]``
+
+-q, --quiet    surpress console output
+-d, --dry-run  dry run, don't commit changes
+
+Examples
+--------
+
+``fix/empty-wheelbarrows``
+    Empties all items, listing all wheelbarrows emptied and their contents
+``fix/empty-wheelbarrows --dry-run``
+    Lists all wheelbarrows that would be emptied and their contents without performing the action.
+``fix/empty-wheelbarrows --quiet``
+    Does the action while surpressing output to console
+``repeat --name empty-wheelbarrows --time 1200 command [ fix/empty-wheelbarrows --quiet ]``
+    Runs empty-wheelbarrows quietly every 1200 game ticks, which is once per in-game day.
```

Comment:
this command can be added to `gui/control-panel` so it can be enabled with a button click (and as I referenced above, it can be set to be enabled by default for new users)

Reply:
Under the maintenance tab I presume?

Reply:
that looks right, though I'd change the `1200` to 
```lua
        command={'--time', '1', '--timeUnits', 'days', '--command', '[', 'fix/stuck-wheelbarrows', '-q', ']'}},
```
for clarity

Reply:
also, once it's integrated in `gui/control-panel`, I'd remove the repeat example here and instead add a note to the introduction text that you can run it automatically by toggling the Maintenance task in `gui/control-panel`.

Reply:
Alrighty, done all this. For anyone else reading this, I did type `fix/stuck-wheelbarrows` instead of `fix/empty-wheelbarrows` in this conversation - I have since noticed and that mistake hasn't carried over to the code.